en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,422 +1,495 @@
 
-- Calculating the modulus of two integer literals.
+    The following operators are available in the expression editor for value comparisons:
+    
+    - `<`: Less than
+    - `>`: Greater than
+    - `<=`: Less than or equal to
+    - `>=`: Greater than or equal to
 
-  ```ballerina
-  10 % 4
-  ```
-  
-- Calculating the modulus of a literal and a variable (`firstNumber`).
+    ###### Examples:
 
-  ```ballerina
-  firstNumber % 100 
-  ```
+    To compare the value of the integer variable `firstNumber` to the integer literal `10`, use one of the following expressions:
+
+    ```ballerina
+    firstNumber > 10
+  
+    firstNumber <= 10
+    ```
+
+    To compare the value of the string variable `greeting` to the string literal `Hi`, use the following expression:
+   
+    ```ballerina
+    greeting == "Hi"    
+    ```
+    To compare an array variable `arrayWithTwoElements` with an array literal `arrayWithOneElement`, use the following expression:
+    
+     ```ballerina
+     arrayWithTwoElements <  arrayWithOneElement
+     ```
+    
+## HTTP 
+
+All the HTTP-related services created in Choreo have a variable called `request` (of type `http:Request`) which developers can use to query information related to the HTTP request. Following is a list of such use cases.
+
+### Read a header value in the request
+A typical HTTP request contains many headers.  These include standard and custom headers. You can use the `request` variable's `getHeader` method to read a specific header value in any expression-enabled input field.
+
+###### Examples:
+
+To read the value of an HTTP header `Access-Control-Allow-Origin`, use the following expression:
+
+```ballerina
+check request.getHeader("Access-Control-Allow-Origin")
+```
   
-- Calculating the modulus of two numerical variables (`firstNumber` and `secondNumber`).
+To read the value of an HTTP header by passing a variable that contains the header name `headerName`, use the following expression:
 
-  ```ballerina
-  firstNumber % secondNumber
-  ```
+```ballerina
+check request.getHeader(headerName)
+```
 
-### Checking The Equality of Basic Types 
+### Check whether a header is present in the request
 
-In the [Ballerina language](https://ballerina.io/), basic types are `string`, `int`, `float`, `decimal`, `boolean`, and `nil`. You can compare the values of these types using the `==` binary operator. The values can come from a literal or a variable. Similarly `!=` operator is used to check the inequality. Note that the resulting value from these operators is always of `boolean` type. 
+Services or integrations in Choreo sometimes need to check the availability of a header before proceeding to something like reading the header value. You can use the  `hasHeader`  method in the `request` object for this purpose.
 
-#### Examples
+###### Examples:
 
-- Checking if two integer literals are equal.
+To check if the HTTP header `content-length` exists given the header name, use the following expression:
```

Comment:
Suggestion: "To check if the HTTP header `content-length` exists in the given header name...."

Reply:
Done